### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by basileLeroy** [Jana2xN/Accessibility#1]
# Feedback
Keyboard navigation looks good! I would just recommend to make the focus styling a little more obvious when something is in focus, the eye should be able to imediately see where your focus is on the page.

The Contrast levels are good as well, but in your nav and footer, your text is slightly outside of the recommended WCAG2.1 contrast ratio.

On your blog page there is a header that is not inside a landmark. each element should be inside landmark (header, nav, main, footer)

All the rest looks good to me!

## Git
I would recommend committing a little more often. it will be a good practice and habit to have towards the future!
